### PR TITLE
chore(frontend): Lemonize `PropertyKeyInfo`

### PR DESCRIPTION
**Pull request by Twixes**
## Problem

This was ugly and used Ant, along with an old version of our logo:

Screenshot 2023-06-30 at 11 20 09

(notice the weird shadow on the left)

## Changes

This is nicer and uses Lemon, along with the current version of our logo:

Screenshot 2023-06-30 at 15 03 22

Also removed all the other obsolete unused SVGs.

### CODE REVIEW

**Review by benjackwhite — APPROVED** [review]
Nice - looks great!

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.0'
+lockfileVersion: '6.1'
```
Comment (thmsobrmlr) [inline pnpm-lock.yaml:1]: Hmm

**Review by thmsobrmlr — APPROVED** [review]
Looks ✨

File: pnpm-lock.yaml (hunk 1 above)
Comment (Twixes) [reply pnpm-lock.yaml]: I'm on pnpm 8.6.0 as specified in package.json and it insists on this, so I guess it's good for committing

**Pull request merged**